foxy/distribution.yaml

```diff
@@ -2094,6 +2094,13 @@ repositories:
         release: release/foxy/{package}/{version}
       url: https://github.com/ros-gbp/ompl-release.git
       version: 1.5.0-1
+  openvslam:
+    release:
```

Comment:
Please add a `source` (and optionally `doc` entry) as well.

Reply:
Added.